### PR TITLE
[bitnami/redmine] Missing env var in cronjob.yaml

### PR DESCRIPTION
**Pull request by Vinche59**
**Description of the change**

mail-receiver-configmap.yaml reference $REDMINE_DATABASE_TYPE variable but this variable is never defined in cronjob.yaml

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by Mauraza** [comment]
Hi @Vinche59,

Did you have any issues?

I deployed a Redmine with `mail-receiver-configmap`  and everything goes well. The environment variables of this configmap are:
```bash
    Environment:
      REDMINE_MAIL_RECEIVER_DB_MYSQL:                 redmain-mariadb
      REDMINE_MAIL_RECEIVER_DB_DATABASE:              my_database
      REDMINE_MAIL_RECEIVER_DB_USERNAME:              bn_redmine
      REDMINE_MAIL_RECEIVER_DB_PASSWORD:              <set to the key 'mariadb-password' in secret 'redmain-mariadb'>  
```

**Comment by Vinche59** [comment]
Hello @Mauraza 

Actually deployment works correctly in my env but when the job execute it failed each time with an error : 
```
"No database settings given. Exiting"
```
In the [receive.sh](https://github.com/bitnami/charts/blob/master/bitnami/redmine/templates/mail-receiver-configmap.yaml) configmap generated, this error is generated because the script can't find ```$REDMINE_DATABASE_TYPE``` env var and I didn't find where this variable should be setted thus this pull request.

But maybe I missed something during the deployment of the helm chart ?

File: bitnami/redmine/templates/cronjob.yaml [hunk 1]
```diff
@@ -84,6 +84,12 @@ spec:
                 {{- end }}
               resources: {{- toYaml .Values.mailReceiver.resources | nindent 16 }}
               env:
+                - name: REDMINE_DATABASE_TYPE
+                {{- if eq .Values.databaseType "mariadb" }}
+                  value: "mariadb"
+                {{- else }}
+                  value: "postgresql"
+                {{- end }}
```
Comment (Mauraza) [inline bitnami/redmine/templates/cronjob.yaml:92]: You can use the same statement as in deployment.yaml
```suggestion
             - name: REDMINE_DATABASE_TYPE
                value: {{ .Values.databaseType | quote }}
```

**Review by Mauraza — COMMENTED** [review]
Hi @Vinche59,

Could you check the comment?

File: bitnami/redmine/templates/cronjob.yaml (hunk 1 above)
Comment (Vinche59) [reply bitnami/redmine/templates/cronjob.yaml]: Hello @Mauraza,

Indeed, it's nicer that way :-)
Modifications done.

**Review by Mauraza — APPROVED** [review]
Hi @Vinche59,

Thanks again for the contribution!! LGTM!

**Pull request merged**